environments/servlet/tests/tomcat/pom.xml

```diff
@@ -176,48 +171,44 @@
         </profile>
 
         <profile>
-            <id>tomcat7</id>
+            <id>tomcat9</id>
             <activation>
                 <property>
-                    <name>tomcat</name>
-                    <value>7</value>
+                    <name>incontainer</name>
                 </property>
             </activation>
-            <dependencies>
 
+            <dependencies>
                 <dependency>
                     <groupId>org.jboss.arquillian.container</groupId>
-                    <artifactId>arquillian-tomcat-embedded-7</artifactId>
+                    <artifactId>arquillian-tomcat-embedded-9</artifactId>
                     <scope>test</scope>
                 </dependency>
-
             </dependencies>
         </profile>
-
         <profile>
-            <id>tomcat8</id>
+            <id>jdk9</id>
             <activation>
-                <property>
-                    <name>!tomcat</name>
-                </property>
+                <jdk>[9,)</jdk>
             </activation>
 
-            <properties>
-                <tomcat.version>${tomcat8.version}</tomcat.version>
-            </properties>
-
-            <dependencies>
-
-                <dependency>
-                    <groupId>org.jboss.arquillian.container</groupId>
-                    <artifactId>arquillian-tomcat-embedded-8</artifactId>
-                    <scope>test</scope>
-                </dependency>
-
-            </dependencies>
-
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludes>
+                                <!-- https://issues.apache.org/bugzilla/show_bug.cgi?id=57314 -->
+                                <exclude>**/ListenerSymmetryTest.java</exclude>
```

Comment:
This should be inspected, why do we still have it excluded and what can we do about it?

Reply:
Also, the `incontainer` profile seems to be excluding it already, is this duplication needed?

Reply:
Looks like it is needed as Maven does not merge the exclusions in 2 profiles; it uses the last one instead.

Reply:
The test exclusion is still needed as related [Tomcat BZ](https://bz.apache.org/bugzilla/show_bug.cgi?id=57314) was closed as WONTFIX. The same test works on other servlet containers (e.g. Jetty) so we probably should not remove it entirely.